CONTRIBUTING.md

```diff
@@ -197,6 +197,19 @@ accepting a fix in that release, i.e. we would very much appreciate someone
 working on it, but are not committing to actively sourcing someone to work on
 it.
 
+GitHub Pull Requests Conventions
+-------------------
+
+Every (non-backport) pull request has to go through a review and get 2 approvals. After
+this is done, the author of the pull request is expected to add any final touches they
+think important and put the `merge me` label on the pull request. If the author lacks
+permissions to apply labels, they should explicitly signal on the discussion thread of the
+pull request.
+
+Currently there is a 2 day buffer period between the last update of a pull request (e.g.
+an addition of the `merge me` label) and the moment the Mergify bot picks up the pull
```

Comment:
No, the commands like refresh/rebase/rebase are a different thing, they work as before.

Reply:
I'm not being clear. I mean, after "mergifyIO rebase", does it wait 2 more days? That's not fatal, just wondering.

Reply:
I think I got you right?.. If you comment on a PR thread `@mergify rebase`, it won't wait before rebasing, because it's a "command" (in Mergify vocabulary), which is not an "action" that Mergify performs on PRs. So, after a comment `@mergify rebase`, Mergify will rebase as before, without waiting 2 days. But it will wait for 2 days before merging the PR (if all other conditions are satisfied).

Reply:
OK, I was interested in the last part of that. If a user, after waiting 2 days, rebases, either with a manual push or with `mergifyIO rebase`, the user may be surprised that extra 2 days of waiting are needed. However, there is usually no good reason to rebase and some of the reasons involve non-trivial changes to CI, github Settings or the code-base, so extra waiting period makes sense.

Reply:
In any case, we now mention "rebase", so the user should not be surprised.